tests/spec/fs.exists.spec.js

```diff
@@ -56,4 +56,28 @@ describe('fs.exists', function() {
       });
     });
   });
+  it('should follow symbolic links and return false if for the resulting path does not exist', function(done) {
```

Comment:
Add a blank line above this, so it visually separates this test from the others.

Reply:
Great question.  To update a PR, you simply `add` and `commit` again on your same `branch`, then `push` your `branch` again to your `origin`.  GitHub will add that commit (or commits) to this PR.

Let me know if you have any trouble or need more help.